server/util/clickhouse/clickhouse.go

```diff
@@ -175,8 +183,27 @@ func ToInvocationFromPrimaryDB(ti *tables.Invocation) *Invocation {
 
 func (h *DBHandle) FlushInvocationStats(ctx context.Context, ti *tables.Invocation) error {
 	inv := ToInvocationFromPrimaryDB(ti)
-	res := h.DB(ctx).Create(inv)
-	return res.Error
+	retrier := retry.New(ctx, &retry.Options{
+		MaxRetries:     numClickhouseRetries,
+		InitialBackoff: 50 * time.Millisecond,
+		MaxBackoff:     3 * time.Second,
+		Multiplier:     2,
+	})
```

Comment:
I'm not sure this buys you much over the stock retrier. I would probably just use that one unless you have a compelling reason not to?

Reply:
Done.